src/main/java/org/cryptomator/ui/preferences/UpdatesPreferencesController.java

```diff
@@ -32,7 +34,10 @@
 @PreferencesScoped
 public class UpdatesPreferencesController implements FxController {
 
-	private static final String DOWNLOADS_URI = "https://cryptomator.org/downloads";
+	private static final String DOWNLOADS_URI_TEMPLATE = "https://cryptomator.org/downloads" //
```

Comment:
```suggestion
	private static final String DOWNLOADS_URI_TEMPLATE = "https://cryptomator.org/downloads/" //
```

Added a slash `/` after `downloads`. At least my browser autocorrects this. There is a whole discussion about it that it's not necessary, but let's just add it. 😂 https://stackoverflow.com/questions/1617058/ok-to-skip-slash-before-query-string